util/TomlChecker.js

```diff
@@ -0,0 +1,30 @@
+import * as TOML from '@ltd/j-toml';
+import * as os from 'os';

```

Comment:
import statements are not consistent with rest of the util files.

Reply:
updated